src/constants/network.js

```diff
@@ -0,0 +1,14 @@
+const NETWORK = {
+  available: {
+    test: {
+      url: 'https://horizon-testnet.stellar.org',
+      useNetworkFunc: 'useTestNetwork',
```

Comment:
Relevant code in js-stellar-base: https://github.com/stellar/js-stellar-base/blob/master/src/network.js

So would it be better to have the `useNetworkFunc: 'useTestNetwork',` instead be `stellarBaseNetworkKey: 'PUBLIC',` (or maybe a shorter name)?